docs/en/ingest-management/fleet/add-fleet-server-kubernetes.asciidoc

```diff
@@ -0,0 +1,392 @@
+       id="production-tab"
+       aria-labelledby="production"
+       hidden="">
+++++
++
+include::add-fleet-server-kubernetes-content.asciidoc[tag=production-secret]
++
+++++
+  </div>
+</div>
+++++
+
+. Save the proposed `Deployment` manifest locally, for example as `fleet-server-dep.yaml`, and adapt it to your needs:
++
+++++
+<div class="tabs" data-tab-group="fleet-k8s">
+  <div role="tablist" aria-label="Fleet Server on Kubernetes">
+    <button role="tab"
+            aria-selected="true"
+            aria-controls="quickstart-dep-tab"
+            id="quickstart-dep">
+      Quick start
+    </button>
+    <button role="tab"
+            aria-selected="false"
+            aria-controls="production-dep-tab"
+            id="production-dep"
+            tabindex="-1">
+      Production
+    </button>
+  </div>
+  <div tabindex="0"
+       role="tabpanel"
+       id="quickstart-dep-tab"
+       aria-labelledby="quickstart-dep">
+++++
++
+include::add-fleet-server-kubernetes-content.asciidoc[tag=quickstart-deployment]
++
+++++
+  </div>
+  <div tabindex="0"
+       role="tabpanel"
+       id="production-dep-tab"
+       aria-labelledby="production-dep"
+       hidden="">
+++++
++
+include::add-fleet-server-kubernetes-content.asciidoc[tag=production-deployment]
++
+++++
+  </div>
+</div>
+++++
+
+. Deploy the configured manifest to create and start the {fleet-server}:
++
+[source, shell]
+------------------------------------------------------------
+kubectl apply -f fleet-server-dep.yaml
+------------------------------------------------------------
++
+[IMPORTANT]
+====
+Ensure the `Deployment` and all referenced `Secrets` are created in the same `Namespace`.
+====
+
+. Check the {fleet-server} Pod logs for errors and confirm in {kib} that the {fleet-server} agent appears as `Connected` and `Healthy` in *{kib} → Fleet*.
++
+[source, shell]
+------------------------------------------------------------
+kubectl logs fleet-server-xxxxx
+------------------------------------------------------------
++
+It could take a while until the {fleet-server} fully starts. If you left the {kib} browser window opened during <<add-fleet-server-kubernetes-preparations>> it will show *Connected* in case everything goes well.
++
+Do not continue with the next steps if the {fleet-server} is not completely up and healthy and appears in Fleet UI.
+
+. Expose the {fleet-server} to {agent}s through the *Fleet Server Host*:
++
+This may include the creation of a Kubernetes `service`, an `ingress` resource, or create DNS registers for FQDNs resolution. There are multiple ways to expose applications in Kubernetes.
++
+The following shows an example to expose the {fleet-server} with a `LoadBalancer` service, which will make the {fleet-server} available for both internal and external clients. Consider creating a `ClusterIP` instead of a `LoadBalancer` service if you only need to expose the {fleet-server} to {agent}s running in the same Kubernetes cluster.
```

Comment:
```suggestion
The following example exposes the {fleet-server} with a `LoadBalancer` service, which will make the {fleet-server} available for both internal and external clients. Consider creating a `ClusterIP` instead of a `LoadBalancer` service if you only need to expose the {fleet-server} to {agent}s running in the same Kubernetes cluster.
```